code-studio/src/js/components/AdvancedShareOptions.jsx

```diff
@@ -94,15 +120,31 @@ var AdvancedShareOptions = React.createClass({
     var optionsNav;
     var selectedOption;
     if (this.state.expanded) {
+      var selectedLiStyle = Object.assign({}, style.nav.li, {color:color.purple});
```

Comment:
Also, have you seen [Radium](https://github.com/FormidableLabs/radium) (which we're already using several places)?